File: api-tester-java/src/main/java/com/revenuecat/apitesterjava/TransactionAPI.java
package com.revenuecat.apitesterjava;

import com.revenuecat.purchases.models.Transaction;

import java.util.Date;

@SuppressWarnings({"unused"})
final class TransactionAPI {
    static void check(final Transaction transaction) {
        final String revenuecatId = transaction.getRevenuecatId();
        final String productId = transaction.getProductId();
        final Date purchaseDate = transaction.getPurchaseDate();
    }
}

File: api-tester-kotlin/src/main/java/com/revenuecat/apitesterkotlin/TransactionAPI.kt
package com.revenuecat.apitesterkotlin

import com.revenuecat.purchases.models.Transaction
import java.util.Date

@Suppress("unused", "UNUSED_VARIABLE")
private class TransactionAPI {
    fun check(transaction: Transaction) {
        val revenuecatId: String = transaction.revenuecatId
        val productId: String = transaction.productId
        val purchaseDate: Date = transaction.purchaseDate
    }
}
